src/main/features/dashboard/index.ts

```diff
@@ -20,6 +21,19 @@ function requestHandler (): express.RequestHandler {
 
 export class DashboardFeature {
   enableFor (app: express.Express) {
+    if (app.settings.nunjucksEnv && app.settings.nunjucksEnv.filters) {
+      app.settings.nunjucksEnv.filters.renderPaymentSchedule = (value: string) => {
```

Comment:
In the future we will start moving these to separate files. Since this is the only one I am not feeling strong for doing it now.

Reply:
sure